src/cookbooks/data_monitoring/deploying_metrics.md

```diff
@@ -0,0 +1,68 @@
+## Deploying Metrics
+To deploy metrics in Bigeye, navigate to a schema, table, or column and click "Add Monitoring." 
+You can choose to add metrics via 4 options
+1. Freshness and Volume (Pipeline Reliability)
+2. Data Quality
+3. All metrics 
+4. Custom SQL, and select the metrics you wish to deploy.
+
+
+![](../../assets/Bigeye/Bigeye-Deploymetrics.png)
+
+![](../../assets/Bigeye/Bigeye-Deploymetrics-2.png)
+
+
+Next, choose the columns to monitor, set schedules, thresholds, and filters, and confirm your selections.
+You can assign the metrics to the relevant collection, grouping related metrics for easier management and monitoring.
+Once deployed, the metrics appear under the Metrics tab on the relevant schema, table, or column page.
+We can backfill the metrics for up to 28 days in the past.
+
+![](../../assets/Bigeye/Bigeye-Deploymetrics-3.png)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/LRhbxFMPTwI?si=yJ98NhvlmGt02eq7" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+
+For more details, please refer to the Bigeye documentation on [how to deploy metrics](https://docs.bigeye.com/docs/deploy-metrics).
+
+Watch the Bigeye tutorial on [how to use the metrics page](https://www.youtube.com/watch?v=jNzSki59AWQ)
+
+### Autometrics
+Autometrics are suggested metrics that monitor anomalies in column-level data, automatically generated for all new datasets in Bigeye. They can be found under the Autometrics tab in the Catalog when viewing a source, schema, table, or column page.
+
```

Comment:
```suggestion

> :warning: **Try to avoid this option!**: On tables with many columns a large number of monitors might get deployed. This increases noise and cost. Instead, it is recommended to [choose relevant metrics from the list of available metrics](#list-of-available-metrics) manually.

```